lua/luasnip/nodes/snippet.lua

```diff
@@ -181,7 +181,7 @@ local function init_snippet_context(context)
 
 	-- context.dscr could be nil, string or table.
 	context.dscr =
-		util.to_line_table(util.wrap_value(context.dscr or context.trigger))
+		util.to_line_table(context.dscr or context.trigger)
```

Comment:
This wasn't supposed to be changed, right?

Reply:
yes `to_line_table` calls wrap_value (now  `to_string_table`) internally already, it was a duplicated call.

Reply:
Ohhh, okay, nvm :+1: